README.md

```diff
@@ -35,6 +35,7 @@ For starter apps and templates, see [create-snowpack-app](./create-snowpack-app)
 ### Dev Environment
 
 - [@snowpack/plugin-dotenv](./plugins/plugin-dotenv)
+- [@snowpack/plugin-typescript](./plugins/plugin-typescript)
```

Comment:
Is this more of a “transform” plugin (not in method called, but to the user)?

Reply:
Since this is only doing type checking, it felt like a "dev environment" plugin. Maybe make this more clear? Maybe add a new "Lint" section?